packages/about/spec/update-manager-spec.js

```diff
@@ -16,17 +16,19 @@ describe('UpdateManager', () => {
 
     it('returns the page for the release when not a dev version', () => {
       expect(updateManager.getReleaseNotesURLForVersion('1.7.0')).toContain(
-        'atom/atom/releases/tag/v1.7.0'
+        'atom-ide-community/atom/releases/tag/v1.7.0'
       );
       expect(updateManager.getReleaseNotesURLForVersion('v1.7.0')).toContain(
-        'atom/atom/releases/tag/v1.7.0'
+        'atom-ide-community/atom/releases/tag/v1.7.0'
       );
       expect(
         updateManager.getReleaseNotesURLForVersion('1.7.0-beta10')
-      ).toContain('atom/atom/releases/tag/v1.7.0-beta10');
+      ).toContain('atom-ide-community/atom/releases/tag/v1.7.0-beta10');
       expect(
         updateManager.getReleaseNotesURLForVersion('1.7.0-nightly10')
-      ).toContain('atom/atom-nightly-releases/releases/tag/v1.7.0-nightly10');
+      ).toContain(
+        'atom-ide-community/atom-nightly-releases/releases/tag/v1.7.0-nightly10'
```

Comment:
Looks like we need to make `v1.7.0`, `v1.7.0-beta10` releases at our main repo, and `v1.7.0-nightly10` and our nightly release repo.

Or just not test against our fork, test against the upstream releases. Maybe any changes to the specs should be reverted?

Testing against our releases makes us fully independent. But if we think the upstream repo will always be available, then it's easier to just test against their releases.

Reply:
Okay, maybe not. The function returns a string, and then there is a text comparison.

This should probably pass CI now.